tests/ffi/common/prepare.sh

```diff
@@ -67,3 +67,9 @@ run_container_in_qm() {
       tail -f /dev/null\""
    exec_cmd "${run_ctr_in_qm}"
 }
+
+init_ffi() {
```

Comment:
Can you call it from a test to show ?

Reply:
Could be fixed for all tests, I assume

Reply:
It succeeds!
Tested on distro: CentOS-Stream-9 auto-osbuild-qemu-rhivos9-qa-regular-aarch64-9235912.5f8b36a9 image.
Trying to pull quay.io/centos-sig-automotive/ffi-tools:latest...
Getting image source signatures
Copying blob sha256:4d0d91f3c28123399dc41254c6c19550e4ac21f7d4108fe05b62288447e977ea
Copying blob sha256:26ef76492da36d9cbdc5cf06b0795e1cb02abe3a7ec52f449aed5d99fb359948
Copying config sha256:226c20eb8a648afbea8807338a88b6ed34d154b01dbcf3d2bac30489a88a5dec
Writing manifest to image destination
226c20eb8a648afbea8807338a88b6ed34d154b01dbcf3d2bac30489a88a5dec
Getting image source signatures
Copying blob sha256:55b06d85b389a2f0ae8f7cca8fea74637be1fd8b3f78edd2993dadef033a83c3
Copying blob sha256:1727e4dc821d9eaaf54a814ca906eb450d24351b74d3c7ff8c63343a804affe3
Copying config sha256:226c20eb8a648afbea8807338a88b6ed34d154b01dbcf3d2bac30489a88a5dec
Writing manifest to image destination
Untagged: quay.io/centos-sig-automotive/ffi-tools:latest
Deleted: 226c20eb8a648afbea8807338a88b6ed34d154b01dbcf3d2bac30489a88a5dec
Getting image source signatures
Writing manifest to image destination
75c857f15cec0d712e175126a0771381e8cd3a5ec281f7ac70ee90bb102cacb9
Container name is: ffi-qm
Container ffi-qm
 status is: running
PASS: The container ffi-qm
 is running
 inside QM.

Reply:
I removed previously commented lines and using now only init_ffi from /common/prepare.sh.
Tested again on TC distro: CentOS-Stream-9 after the changes-PASS.